airflow/www/templates/airflow/trigger.html

```diff
@@ -34,13 +35,19 @@ <h2>Trigger DAG: {{ dag_id }}</h2>
     <input type="hidden" name="csrf_token" value="{{ csrf_token() }}">
     <input type="hidden" name="dag_id" value="{{ dag_id }}">
     <input type="hidden" name="origin" value="{{ origin }}">
-    <div class="form-group">
       <label for="conf">Configuration JSON (Optional)</label>
       <textarea class="form-control" name="conf" id="json">{{ conf }}</textarea>
     </div>
     <p>
       To access configuration in your DAG use <code>{{ '{{ dag_run.conf }}' }}</code>.
     </p>
+    <div class="form-group">
+      <label class="switch-label">
+        <input class="switch-input" name="unpause" id="unpause" type="checkbox" checked>
+        <span class="switch" aria-hidden="true"></span>
+        Unpause DAG when triggered
+      </label>
+    </div>
```

Comment:
Should we only show this if the dag is currently paused?

Reply:
Currently, trigger.html only has access to `dag_id`, but nothing about the dag itself.